src/dev/typescript/run_type_check_cli.ts

```diff
@@ -69,7 +69,11 @@ export async function runTypeCheckCli() {
     process.exit();
   }
 
-  await buildAllTsRefs(log);
+  const { failed } = await buildAllTsRefs(log);
+  if (failed) {
+    log.error('Unable to build TS project refs');
```

Comment:
I think we should use `throw createFailError('Unable to build TS project refs')` to get similar behavior but properly trigger shutdown by using error propagation.

Reply:
Oh, this CLI isn't implemented with `run()` from `@kbn/dev-utils` so that's not going to work. NM I guess.